dlp/src/main/java/dlp/snippets/InspectStringMultipleRules.java

```diff
@@ -0,0 +1,143 @@
+ */
+
+package dlp.snippets;
+
+// [START dlp_inspect_string_multiple_rules]
+
+import com.google.cloud.dlp.v2.DlpServiceClient;
+import com.google.privacy.dlp.v2.ByteContentItem;
+import com.google.privacy.dlp.v2.ByteContentItem.BytesType;
+import com.google.privacy.dlp.v2.ContentItem;
+import com.google.privacy.dlp.v2.CustomInfoType.DetectionRule.HotwordRule;
+import com.google.privacy.dlp.v2.CustomInfoType.DetectionRule.LikelihoodAdjustment;
+import com.google.privacy.dlp.v2.CustomInfoType.DetectionRule.Proximity;
+import com.google.privacy.dlp.v2.CustomInfoType.Dictionary;
+import com.google.privacy.dlp.v2.CustomInfoType.Dictionary.WordList;
+import com.google.privacy.dlp.v2.CustomInfoType.Regex;
+import com.google.privacy.dlp.v2.ExclusionRule;
+import com.google.privacy.dlp.v2.Finding;
+import com.google.privacy.dlp.v2.InfoType;
+import com.google.privacy.dlp.v2.InspectConfig;
+import com.google.privacy.dlp.v2.InspectContentRequest;
+import com.google.privacy.dlp.v2.InspectContentResponse;
+import com.google.privacy.dlp.v2.InspectionRule;
+import com.google.privacy.dlp.v2.InspectionRuleSet;
+import com.google.privacy.dlp.v2.Likelihood;
+import com.google.privacy.dlp.v2.LocationName;
+import com.google.privacy.dlp.v2.MatchingType;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+
+public class InspectStringMultipleRules {
+
+  public static void inspectStringMultipleRules() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String textToInspect = "patient: Jane Doe";
+    inspectStringMultipleRules(projectId, textToInspect);
+  }
+
+  // Inspects the provided text, avoiding matches specified in the exclusion list.
+  public static void inspectStringMultipleRules(String projectId, String textToInspect)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DlpServiceClient dlp = DlpServiceClient.create()) {
+      // Specify the type and content to be inspected.
+      ByteContentItem byteItem =
+          ByteContentItem.newBuilder()
+              .setType(BytesType.TEXT_UTF8)
+              .setData(ByteString.copyFromUtf8(textToInspect))
+              .build();
+      ContentItem item = ContentItem.newBuilder().setByteItem(byteItem).build();
+
+      // Construct hotword rules
+      HotwordRule patientRule = HotwordRule.newBuilder()
+          .setHotwordRegex(Regex.newBuilder()
+              .setPattern("patient"))
+          .setProximity(Proximity.newBuilder()
+              .setWindowBefore(10))
+          .setLikelihoodAdjustment(LikelihoodAdjustment.newBuilder()
+              .setFixedLikelihood(Likelihood.VERY_LIKELY))
+          .build();
```

Comment:
Is this formatted correctly?  Shouldn't they be aligned to make reading easier?

Reply:
Better yet, a single line sections
```
      HotwordRule patientRule = HotwordRule.newBuilder()
          .setHotwordRegex(Regex.newBuilder().setPattern("patient"))
          .setProximity(Proximity.newBuilder().setWindowBefore(10))
          .setLikelihoodAdjustment(LikelihoodAdjustment.newBuilder().setFixedLikelihood(Likelihood.VERY_LIKELY))
          .build();
```
Or is that too wide?

Reply:
The autoformat breaks the setLikelihoodAdjustment line because it's too long:

```
      HotwordRule patientRule = HotwordRule.newBuilder()
          .setHotwordRegex(Regex.newBuilder().setPattern("patient"))
          .setProximity(Proximity.newBuilder().setWindowBefore(10))
          .setLikelihoodAdjustment(
              LikelihoodAdjustment.newBuilder().setFixedLikelihood(Likelihood.VERY_LIKELY))
          .build();

```

I found it easier to read the structure when the other lines were also broken.

The linter / autoformatter seems happy either way, and I didn't see anything in the style guide that was strongly opinionated about it, so I went with what seemed easier to read. 

LMK if you still want this changed, I don't feel strongly about it and don't mind switching if my instincts on what's easier to read seem off.